src/main/java/com/practicesoftwaretesting/LoginUserPayload.java

```diff
@@ -0,0 +1,12 @@
+package com.practicesoftwaretesting;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
Since there is only two fields Builder is not necessary here. Just compare what do you like more Builder or Constructor
```
        LoginUserPayload userSignIn = LoginUserPayload.builder()
                .email("220john221@dou.example")
                .password("12Example#")
                .build();
```

```
          LoginUserPayload userSignIn = new LoginUserPayload("220john221@dou.example", "12Example#")
```